src/main/java/org/folio/circulation/domain/subscribers/LoanRelatedFeeFineClosedEvent.java

```diff
@@ -0,0 +1,21 @@
+package org.folio.circulation.domain.subscribers;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import io.vertx.core.json.JsonObject;
+
+public class LoanRelatedFeeFineClosedEvent {
```

Comment:
Is the ID of the fee / fine not relevant?

Reply:
No, it is never used in circulation.

Reply:
Ok, given circulation is the only consumer of this message, that makes it's conclusion a little odd. I'm guessing it was done to try to make the event more general?

Reply:
Yes, correct.

Reply:
Ok, I'll resolve this